src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/model/Booking.kt

```diff
@@ -12,7 +12,6 @@ data class Booking(
   val returnToCustodyDate: LocalDate? = null,
   val fixedTermRecallDetails: FixedTermRecallDetails? = null,
   val bookingId: Long = -1L,
-  val calculateErsed: Boolean = false,
```

Comment:
after I did all that spelling correction ;)

Reply:
lol sorry